pkg/operator/apis/monitoring/v1/operator_config.go

```diff
@@ -70,6 +70,51 @@ func (c *CollectionSpec) ScrapeConfigs() ([]*promconfig.ScrapeConfig, error) {
 			SourceLabels: prommodel.LabelNames{"__meta_kubernetes_node_name"},
 			TargetLabel:  "node",
 		},
+		// Capture controller name
```

Comment:
Nice! As per the comment I left on Bartek's thread, let's move this and make it conditional (and therefore backwards-compatible).